models/matrice_100/matrice_100.sdf.jinja

```diff
@@ -293,6 +293,8 @@
       <hil_state_level>false</hil_state_level>
       <opticalFlowSubTopic>/px4flow/link/opticalFlow</opticalFlowSubTopic>
       <lidarSubTopic>/sf10a/link/lidar</lidarSubTopic>
+      <enable_lockstep>true</enable_lockstep>
+      <use_tcp>true</use_tcp>
```

Comment:
Do we want TCP and lockstep to be true by default?

Reply:
I'd say lockstep yes. TCP can be discussed, I have no strong opinion and @bkueng is right that it's not necessarily needed because we wait anyway for messages from the other side, so buffers won't fill up and we won't have any drops as long as everything is on localhost. Once we use SITL over the network, we should use TCP.